package/MDAnalysis/coordinates/memory.py

```diff
@@ -264,6 +264,14 @@ def __init__(self, coordinate_array, order='fac',
         super(MemoryReader, self).__init__()
         self.filename = filename
         self.stored_order = order
+
+        try:
```

Comment:
add a comment that references #1685 and explains what the intention of the code is